Added DB sessions config

jeffturcotte opened the pull request.

jeffturcotte committed:
Added DB sessions config

jeffturcotte committed:
Added .user.ini that specifies proper session cleanup values

kerrig commented:
I meant to comment on this last week but totally forgot. We should also have the database table added and committed as well before we add the config for it.

jeffturcotte committed:
Updated Craft to 3.4
Updated plugins.
Added latest DB dump.

jeffturcotte commented:
Added a few more commits (some being tangentially relevant)

- Upgraded to Craft 3.4 to get the session table creation cli script.
- Ended up updating all other plugins as well
- Added .user.ini with proper session GC settings
- Added new sql dump of upgraded DB w/ session table.
